pkg/tracing/tracing_test.go

```diff
@@ -65,7 +65,7 @@ func TestOnStoreWithEnabled(t *testing.T) {
 		Endpoint: "test-endpoint",
 	}
 
-	tp.OnStore(zap.NewExample().Sugar())("config-tracing", cfg)
+	tp.OnStore(zap.NewExample().Sugar(), nil)("config-tracing", cfg)
```

Comment:
It would be good to add test coverage for the non-nil case too

Reply:
Sure. this is just a bandaid to be able to build and verify the changes for now